src/page-building/landingpage/career.tsx

```diff
@@ -0,0 +1,34 @@
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+import { TeaserGrid } from '../../components/teasers/teaser-grid';
+import { Teaser } from '../../components/teasers/teaser';
+import React from 'react';
+import { SyPersonioJob } from '../../@types/personio';
+import { HomePageHeaderBlock } from './support';
+
+interface CareerProps {
+  positions: SyPersonioJob[];
+}
+
+export const Career = ({ positions }: CareerProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <HomePageHeaderBlock
+        topline={t('main.career.kicker')}
+        headline={t('main.career.title')}
+        large={true}
+      >
+        {t('main.career.text')}
+      </HomePageHeaderBlock>
+
+      <TeaserGrid>
+        {positions.map((item) => (
+          <Teaser key={item.id} title={item.name} linkTo={item.fields?.path}>
+            {item.short}
+          </Teaser>
+        ))}
+      </TeaserGrid>
+    </>
+  );
+};
```

Comment:
This section existed before inline and I created a separate file now side by side to the stories.